.changeset/config.json

```
@@ -3,7 +3,7 @@
   "changelog": "@changesets/cli/changelog",
   "commit": false,
   "fixed": [],
-  "linked": [],
+  "linked": [["@whereby.com/browser-sdk", "@whereby.com/core", "@whereby.com/media"]],
```

```suggestion
  "linked": [],
```

This makes all packages bump to the same version. I tested the PR as it is now:
- Bumped media with a major release
- `yarn changeset version`
- browser-sdk, core, and media all gets bumped to version `3.0.0`

This is not what we want. Tested again with the change I'm suggesting here:
- Bumped media with a major release
- `yarn changeset version`
- media is bumped to `2.0.0`
- core is bumped to `0.12.2`
- browser-sdk is bumped to `2.8.2`
- core dependency is bumped in browser-sdk, and media dependency is bumped in core.

This is the behaviour that we want, so we need to remove this change here

---

Ah yes. We do not need to link these packages in the changeset config 👍 

Updated in the PR now (rebased)  ✨